config/default_records/submission_templates/010_bait_capture_library_submission_templates.yml

```diff
@@ -0,0 +1,9 @@
+# This submission template is associated with the Limber Bait Capture Library preparation pipeline.
+---
+Limber-Htp - RVI - Bait Capture Library:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_bait_capture_library_prep", "limber_bait_capture_library_prep_multiplexing"]
+    project_name: Respiratory Virus and Microbiome Initiative
```

Comment:
Just checking, is this the official project name? Does this Project exist in production?
If we also know the Study, and that is fixed, it can be added here too I think.
It may be for RVI these might vary and we need to allow them to specify these on the submissions rather than hardcode here, we can ask R&D